hubgrep/cli_blueprint/import_data.py

```diff
@@ -29,10 +34,67 @@ def add_hoster(hoster_dict: dict):
     return hosting_service
 
 
-def fetch_hoster_repos(export_url, hoster):
+def append_repos(gz_file, hoster, new_table_name):
```

Comment:
maybe the most interesting new function:
- creates a temporary table for just this one export
- unpacks the export csv to this temp table
- appends the temp table to the persistent table, containing all the repo data, adding the hoster_id in the same query
- delete the temp table